spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,63 @@
+
+
+
+describe 'User Stories' do
+  let(:takeaway) { Takeaway.new }
+  let(:menu) { Menu.new}
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+
+  context '#menu' do
+
+
+    it 'displays a menu with a list of dishes with prices' do
+      expect(menu.dishes).to eq ({
+        :salad => 12,
+        :burger => 14,
+        :fajita => 13,
+        :pasta => 8,
+        :brownies => 9
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)